java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/build/preprocessor/maven/MavenPrepareCommand.java

```diff
@@ -87,12 +89,26 @@ private void handleBuild(Path file, boolean topLevel) throws IOException {
         }
     }
 
-    private boolean handlePlugins(List<Plugin> plugins, boolean pluginManagement, boolean topLevel) {
+    private boolean handlePlugins(List<Plugin> plugins, boolean pluginManagement, boolean topLevel) throws IOException {
         boolean modified = false;
+        BuildRecipeInfo info = BuildRecipe.BUILD.getHandler().parse(Path.of(PLUGIN_INFO + "/maven.yaml"));
```

Comment:
This is the preprocessor command, you need to use the disabledPlugins command line argument that was passed in, maven.yaml is not present here.